reference/tools/gnu/autotools.rst

```diff
@@ -0,0 +1,111 @@
+.. _conan_tools_gnu_build_helper:
+
+Autotools
+=========
+
+
+The ``Autotools`` build helper is a wrapper around the command line invocation of autotools. It will abstract the
+calls like ``./configure`` or ``make`` into Python method calls.
+
+Usage:
+
+.. code:: python
+
+    from conans import conanfile
+    from conan.tools.gnu import Autotools
+
+    class App(ConanFile):
+        settings = "os", "arch", "compiler", "build_type"
+
+        def build(self):
+            autotools = Autotools(self)
+            autotools.configure()
+            autotools.make()
+
+It will read the ``conanbuild.conf`` file generated by the :ref:`AutotoolsToolchain<conan_tools_gnu_autotools_toolchain>`
+to know read the arguments for calling the configure and make scripts:
+
+- **configure_args**: Arguments to call the ``configure`` script.
+- **make_args**: Arguments to call the ``make`` script.
+
+
+Reference
+---------
+
+.. currentmodule:: conan.tools.gnu.autotools
+
+.. autoclass:: Autotools
+    :members:
+
+
+A note about relocatable shared libraries in macOS built the  Autotools build helper
```

Comment:
I think I would make this its own page, and linked it from here. Not sure where, but it feels a bit too much here (can stay here, if we don't find a better place)

Reply:
I agree, but let's agree first where.